frontend/src/metabase/visualizations/components/Visualization.jsx

```diff
@@ -396,6 +396,7 @@ export default class Visualization extends Component {
                         visualizationIsClickable={this.visualizationIsClickable}
                         onRenderError={this.onRenderError}
                         onRender={this.onRender}
+                        onActionDismissal={() => this.setState({ clicked: null })}
```

Comment:
We should try to avoid arrow functions inside `render()` for performance reasons so move `() => ...` to a class method! See https://flexport.engineering/optimizing-react-rendering-part-1-9634469dca02